### DISCUSSION TITLE
Remember settings

### DISCUSSION THREAD

**Discussion by okanIz** [Sapd/HeadsetControl#187] [Scripts]
Is it somehow possible to save the configuration? 
Whenever i turn off the headset, the next day the config will be overwritten by the default.

I own the Logitech 733 Headset and want to save the following setting **headsetcontrol -l 0**

**Comment by Sapd** [comment]
You need an additional script like this: https://github.com/Sapd/HeadsetControl/issues/123#issuecomment-752710660
